Kernel/linker_amd64.ld

```diff
@@ -0,0 +1,90 @@
+ENTRY(init)
+
+KERNEL_VIRTUAL_BASE = 0x2000000000;
```

Comment:
we might want to change this value in the future gcc has
```
-mcmodel=kernel

    Generate code for the kernel code model. The kernel runs in the negative 2 GB of the address space. This model has to be used for Linux kernel code.

```
for example, I could not find a clang eqivalent though (they have -mkernel though, but dont say what it does)